packages/webapp/pages/posts/[id].tsx

```diff
@@ -100,8 +99,8 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
     scrollProperty: 'scrollY',
   });
 
-  if (!initialData && (isFallback || isLoading)) {
```

Comment:
I think that the solution to the problem should be this:
`if (!initialData && (isFallback || !isFetched))`

Reply:
Yep, worked like a charm! Let me push it quickly!